crowdfunding/users/serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+from .models import CustomUser
+# use of Model Serializer to save time on setting password
+
+from rest_framework import permissions
+
+#custom permissions for project owner only to be able to edit
+class IsOwnerOrReadOnly(permissions.BasePermission):
+
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS: #is the logged in user the owner?
+            return True
+        return obj.owner == request.user #if it's true - the they should be able to edit 
+
+class IsSupporterOrReadOnly(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        return obj.supporter == request.user
+
+class IsOwnersProfile(permissions.BasePermission):
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        return obj.id == request.user.id
```

Comment:
a bunch of good permissions here, but shouldn't they be in a permissions.py file, rather than in serializers.py?